.github/workflows/cleanup.yml

```diff
@@ -24,24 +23,22 @@ on:
     paths:
       - 'tool.yaml'
       - 'iam.yaml'
-
 concurrency:
   group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}-aod-cleanup'
   cancel-in-progress: true
-
 jobs:
   cleanup:
     permissions:
       contents: 'write'
       id-token: 'write'
       pull-requests: 'write'
-    uses: 'abcxyz/access-on-demand/.github/workflows/cleanup.yml@v0.1.9' # Change here to pin the tag or commit.
+    uses: 'abcxyz/access-on-demand/.github/workflows/cleanup.yml@625f685a62f42ddfa48c8c6da767ae8a0961a2e8' # Change here to pin the tag or commit. ratchet:abcxyz/access-on-demand/.github/workflows/cleanup.yml@v0.1.10
```

Comment:
I would remove the "change here..." text and document how to use `ratchet upgrade` ([doc](https://github.com/sethvargo/ratchet?tab=readme-ov-file#upgrade)) in the readme

Reply:
ack, will post a follow up.